assets/src/edit-story/elements/sticker/stickers/sampleSticker.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+const SampleSticker = ({ style }) => (
+  <svg
```

Comment:
Any reason why this is not an actual SVG file?

Reply:
I was getting issues where it was pulling it in as a dataURI instead of a react component when I did that. Also figured we may want to disable SMIL animations or something embedded in svg for the layers panel version of it and having it as a react component allows us to have that control

Reply:
dataURI is off the table now. As for SMIL, do we plan on supporting that?

I was mainly asking because we want to make it as easy as possible to add new stickers. And just dropping a file into a folder is as easy as it gets.